msteams-platform/graph-api/meeting-transcripts/get-transcripts-api.md

```diff
@@ -0,0 +1,883 @@
+                "joinWebUrl": "https://teams.microsoft.com/l/meetup-join/19%3ameeting_MTM5OTY3MGUtNmY4Mi00Yjg4LTk2MDUtY2IyZGRlNmU1ZjA2%40thread.v2/0?context=%7b%22Tid%22%3a%222432b57b-0abd-43db-aa7b-16eadd115d34%22%2c%22Oid%22%3a%2214b779ae-cb64-47e7-a512-52fd50a4154d%22%7d",
+                "chatInfo": {
+                    "threadId": "19:meeting_MTM5OTY3MGUtNmY4Mi00Yjg4LTk2MDUtY2IyZGRlNmU1ZjA2@thread.v2",
+                    "messageId": "0",
+                    "replyChainMessageId": null
+                }
+            }
+        ]
+    }
+    ```
+
+    </details>
+
+    > [!NOTE]
+    > The response payload also contains the `threadID` member in the `chatInfo` property. Your app can use it for obtaining meeting ID if it's subscribed to receive user-level notifications.
+
+4. **Fetch transcript**: The organizer ID and meeting ID obtained in the Steps 2 and 3 let your app fetch the transcripts for that particular meeting event.
+
+    To fetch transcripts, you'll need to:
+
+    1. **Retrieve transcript ID based on organizer ID and meeting ID**:
+
+       Use the following example to request the transcript ID:
+
+        ```http
+        GET https://graph.microsoft.com/beta/users('14b779ae-cb64-47e7-a512-52fd50a4154d')/onlineMeetings/('MSoxNGI3NzlhZS1jYjY0LTQ3ZTctYTUxMi01MmZkNTBhNDE1NGQqMCoqMTk6bWVldGluZ19ObVUwTlRreFl6TXRNMlkyTXkwME56UmxMV0ZtTjJZdE5URmlNR001T1dNM1pqWTJAdGhyZWFkLnYy')/transcripts
+        ```
+
+        In this example:
+
+        - The meeting ID is included as the value for `onlineMeetings`: *MSoxNGI3NzlhZS1jYjY0LTQ3ZTctYTUxMi01MmZkNTBhNDE1NGQqMCoqMTk6bW
+    VldGluZ19ObVUwTlRreFl6TXRNMlkyTXkwME56UmxMV0ZtTjJZdE5URmlNR001T1dNM
+    1pqWTJAdGhyZWFkLnYy*.
+        - The organizer ID is *14b779ae-cb64-47e7-a512-52fd50a4154d*.
+
+        The response payload contains the transaction ID for the meeting ID and organizer ID in the `id` member of the `value` property.
+        <br>
+        <details>
+        <summary><b>Example</b>: Response payload for getting transaction ID</summary>
+
+        ```json
+        {
+            "@odata.context": "https://graph.microsoft.com/beta/$metadata#users('14b779ae-cb64-47e7-a512-52fd50a4154d')/onlineMeetings('MSoxNGI3NzlhZS1jYjY0LTQ3ZTctYTUxMi01MmZkNTBhNDE1NGQqMCoqMTk6bWVldGluZ19ObVUwTlRreFl6TXRNMlkyTXkwME56UmxMV0ZtTjJZdE5URmlNR001T1dNM1pqWTJAdGhyZWFkLnYy')/transcripts",
+            "@odata.count": 1,
+            "value": [
+                {
+                    "id": "MSMjMCMjMDEyNjJmNjgtOTc2Zi00MzIxLTlhNDQtYThmMmY4ZjQ1ZjVh",
+                    "createdDateTime": "2022-04-14T11:34:39.5662792Z"
+                }
+            ]
+        }
+        ```
+
+        In this example, the transaction ID is *MSMjMCMjMDEyNjJmNjgtOTc2Zi00MzIxLTlhNDQtYThmMmY4ZjQ1ZjVh*.
+
+        </details>
+
+    1. **Access and get meeting transcript based on the transcript ID**:
+
+        Use the following example to request the transcripts for a specific meeting in the `.vtt` format:
```

Comment:
'request the transcript content' since we're fetching content of a single transcript?